spec/features/admin/applicants/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'the admin/applicants' do
+  before(:each) do
+    @bob = Applicant.create!(name: "Bob", street_address: "1234 Bob's Street", city: "Fudgeville", state: "AK", zip_code: 27772, description: "I like petz", application_status: "Pending")
+    @shelter1 = Shelter.create!(name: 'Aurora Shelter', city: 'Aurora, CO', foster_program: false, rank: 9)
+    @pet_1 = Pet.create!(adoptable: true, age: 1, breed: "sphynx", name: "Lucille Bald", shelter_id: @shelter1.id)
+    ApplicantsPet.create!(applicant: @bob, pet: @pet_1)
+  end
+
+  describe "As a visitor" do
+    describe "When I visit the admin applicants index page" do
+      it "displays all applicants who have applications submitted and their attributes" do
+        visit "/admin/applications"
+
+        expect(page).to have_link("Applicants Show page")
```

Comment:
Using `within` blocks is a great way to make your tests even more robust. If we think about being on an index page, we are getting all of our objects displayed. So by using `within` blocks we can really isolate where something is on the page and expect that section to not have things as well.